pulseguardian/management.py

```diff
@@ -9,106 +9,106 @@
 
 import requests
 
+from pulseguardian import config
 MAX_RETRY = 5
 
 
 class PulseManagementException(Exception):
     pass
 
+"""
+Wrapper around the RabbitMQ management plugin's REST API.
+"""
```

Comment:
Yeah, that was meant to describe the whole lib, actually.  Not a particular function.  So, from my internetting, it looks like the appropriate thing for a module is to make it the first statement.